v0/destinations/fb_custom_audience/data/eventConfig.json

```diff
@@ -6,48 +6,5 @@
             "properties.isRaw"
```

Comment:
Let's omit this for setting isRaw let's not use `constructPayload` as for boolean falsy value it will always return `undefined` instead of `false`

Reply:
added meta data support